src/ast/mod.rs

```diff
@@ -551,12 +551,34 @@ pub enum Statement {
     Rollback { chain: bool },
     /// CREATE SCHEMA
     CreateSchema { schema_name: ObjectName },
-
     /// ASSERT <condition> [AS <message>]
     Assert {
         condition: Expr,
         message: Option<Expr>,
     },
+    /// DEALLOCATE [ PREPARE ] { name | ALL }
+    ///
+    /// Note: this is a PostgreSQL-specific statement.
+    Deallocate {
+        name: Option<ObjectName>,
+        all: bool,
```

Comment:
You shouldn't need to handle `ALL` separately, from the parser viewpoint, it's as good an object name, as any. Can `name` be schema-qualified? The docs are not clear on that.

Reply:
Sorry. I tried to check and test again. It is not schema-qualified. I will remove `all` and fix type of `name` to an `Ident`.